packages/components/src/list/style.scss

```diff
@@ -103,7 +143,7 @@ a.woocommerce-list__item {
 
 	&.is-complete {
 		.woocommerce-task__icon {
-			background-color: var(--wp-admin-theme-color);
+			background-color: var( --wp-admin-theme-color );
```

Comment:
My editor does this as well, adding spaces on auto format, but it appears our linter doesn't like this and prefers no spaces.
It be nice to get to a consensus, between the two, I guess it hasn't quite driven me crazy enough to update it yet (just throwing it out, no need to fix that as part of this).

Reply:
@louwie17 whoops, this will fail the css lint anyway. I'll fix it. We could add a prettierignore but, actually we should just add some prettier option overrides for the bracket spacing because I *think* thats the only conflict with stylelint